include/wx/bmpbndl.h

```diff
@@ -76,6 +76,9 @@ class WXDLLIMPEXP_CORE wxBitmapBundle
     // This overload currently makes a copy of the data.
     static wxBitmapBundle FromSVG(const char* data, const wxSize& sizeDef);
 
+    // This overload can work on non 0 terminted data
+    static wxBitmapBundle FromSVG(const unsigned char* data, size_t len, const wxSize& sizeDef);
```

Comment:
Would it make sense to use `wxByte` here for clarity? I.e.
```suggestion
    static wxBitmapBundle FromSVG(const wxByte* data, size_t len, const wxSize& sizeDef);
```